.github/workflows/datasets.yml

```diff
@@ -0,0 +1,46 @@
+name: Flower Baselines
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "datasets/flwr_datasets/**"
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "datasets/flwr_datasets/**"
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

Reply:
@tanertopal Why did you remove the `datasets/flwr_datasets/**`. I thought to run the tests only when there was any change in the library itself

Reply:
Is their no cross depend code? Meaning if anything changes in Flower does that have any influence on datasets?

Reply:
On the Flower's side, there'll be examples that install the Flower Datasets library (the dependency is added to the pyproject.toml). They are already implemented - it's just a modification to the existing partition code of our current examples. It can be merged when the library is released. 
On the Flower Datasets's side, I don't think we should add such tests (there will be tests that for direct ML frameworks integration).
To sum up, there should not be any case that the change of the code in one library (without a release) affects the other library.